.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line no-undef
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
- [ ] Kindly use the same eslint config rules stated [here](https://github.com/microverseinc/linters-config), as it keeps the project professional.

Reply:
oh! Thanks! I forgot to change those! 😣